Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Input/OculusQuestInputManager.cs

```diff
@@ -158,6 +159,7 @@ public override void Disable()
 
             RemoveAllControllerDevices();
             RemoveAllHandDevices();
+            MRTKOculusConfig.Instance.OnCustomHandMaterialUpdate -= UpdateHandMaterial;
```

Comment:
Id remove the listener at the top to be safe

Reply:
Are you pointing me to move Line 162 up to Line 159?  Or just eliminating the unsubscription to the event. 
Can you describe it?

Reply:
Sorry I hadn't seen your reply.

My idea is just to move the unsub part a little higher in the function call, to make sure the removing calls don't somehow trip the callback.